elitea_test/test_client.py

```diff
@@ -0,0 +1,195 @@
+import pytest
+from unittest.mock import Mock, patch
+from reportportal_client import RPClient, OutputType, StepReporter, Issue
+from requests import Session
+from requests.adapters import HTTPAdapter
+
+@pytest.fixture
+def rp_client():
+    return RPClient(
+        endpoint="http://example.com",
+        project="test_project",
+        api_key="test_api_key",
+        log_batch_size=20,
+        is_skipped_an_issue=True,
+        verify_ssl=True,
+        retries=3,
+        max_pool_size=50,
+        launch_uuid="test_launch_uuid",
+        http_timeout=(10, 10),
+        log_batch_payload_size=1024,
+        mode="DEFAULT",
+        launch_uuid_print=False,
+        print_output=OutputType.STDOUT
+    )
+
+def test_launch_uuid(rp_client):
+    assert rp_client.launch_uuid == "test_launch_uuid"
+
+def test_endpoint(rp_client):
+    assert rp_client.endpoint == "http://example.com"
+
+def test_project(rp_client):
+    assert rp_client.project == "test_project"
+
+def test_step_reporter(rp_client):
+    assert isinstance(rp_client.step_reporter, StepReporter)
+
+def test_init_session(rp_client):
+    rp_client._RPClient__init_session()
+    assert isinstance(rp_client.session, Session)
+    assert rp_client.session.headers['Authorization'] == 'Bearer test_api_key'
```

Comment:
These tests are OK, but have no much value, since it verifies setting class fields in constructor.